examples/udp_example.rs

```diff
@@ -0,0 +1,272 @@
+/* ICMP example */
+
+use clap::{arg, Command};
+use num_derive::{FromPrimitive, ToPrimitive};
+use packetvisor::pv;
+use pnet::{
+    datalink::{interfaces, MacAddr, NetworkInterface},
+    packet::MutablePacket,
+    packet::{
+        arp::{ArpHardwareTypes, ArpOperations, MutableArpPacket},
+        ethernet::{EtherTypes, MutableEthernetPacket},
+    },
+};
+use signal_hook::SigId;
+use std::{
+    env,
+    io::Error,
+    net::Ipv4Addr,
+    sync::atomic::{AtomicBool, Ordering},
+    sync::Arc,
+};
+
+#[derive(ToPrimitive, FromPrimitive)]
+enum Protocol {
+    ARP = 1,
+    UDP = 2,
+    OTHER = 3,
+}
```

Comment:
same opinion in https://github.com/tsnlab/packetvisor/pull/50

I thought enum Protocol { ARP, ICMP, OTHER } will work but I don't know the elements in enum need to have integer.
Could you check if they should have integer?